nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/pom.xml

```diff
@@ -108,7 +108,7 @@
         <nifi.provenance.repository.buffer.size>100000</nifi.provenance.repository.buffer.size>
 
         <!-- Component status repository properties -->
-        <nifi.components.status.repository.implementation>org.apache.nifi.controller.status.history.VolatileComponentStatusRepository</nifi.components.status.repository.implementation>
+        <nifi.components.status.repository.implementation>org.apache.nifi.controller.status.history.questdb.EmbeddedQuestDbStatusHistoryRepository</nifi.components.status.repository.implementation>
```

Comment:
I recommend reverting this particular change for this pull request. There is a good deal to review, so if we get to the place where this is ready to be the default setting, we should consider that after reviewing the substantive changes.